2014 Primary Election Calaveras test failure

Missing write-in candidate. Can't find any docs from the county that contains precinct-level or county-level write-in results.

```
E           AssertionError: Calaveras failed
E           assert {('"Bo" Bogda...'): 2262, ...} == {('"Bo" Bogdan...'): 2262, ...}
E             Omitting 65 identical items, use -v to show
E             Left contains more items:
E             {('Patrick D. Hogan', 'Calaveras', 5.0, 'State Assembly'): 4}
E             Use -v to get the full diff
```

--- a/2014/20140603__ca__primary__calaveras__precinct.csv
+++ b/2014/20140603__ca__primary__calaveras__precinct.csv
@@ -1889,3 +1889,4 @@ Calaveras,570,Treasurer,,DEM,John Chiang,152
 Calaveras,570,U.S. House,4,REP,"Arthur ""Art"" Moore",65
 Calaveras,570,U.S. House,4,IND,Jeffrey D. Gerlach,70
 Calaveras,570,U.S. House,4,REP,Tom McClintock,230
+Calaveras,Write-In,State Assembly,5,LIB,Patrick D. Hogan,4

--- a/src/swdb/writeins.py
+++ b/src/swdb/writeins.py
@@ -30,5 +30,16 @@ def amador_p14(_):
              'precinct': 'Write-In',
              'votes': 5}]
 
-WRITEINS = {'P14': {'Alameda': alameda_p14,
-                    'Amador':  amador_p14}}
+
+def calaveras_p14(_):
+    return [{'candidate': 'Patrick D. Hogan',
+             'county': 'Calaveras',
+             'office': 'State Assembly',
+             'district': '5',
+             'party': 'LIB',
+             'precinct': 'Write-In',
+             'votes': 4}]
+
+WRITEINS = {'P14': {'Alameda':   alameda_p14,
+                    'Amador':    amador_p14,
+                    'Calaveras': calaveras_p14}}

